create_json.hpp

```diff
@@ -0,0 +1,58 @@
+
+///////////////////////////////////////////////////////////////////////////////
+// create_json.hpp
+//
+// Creates JSON file for unit tests output
+//
+// Author: Namanh Tran (namanhtran@csu.fullerton.edu)
+// Last Modified: Apirl 1, 2020
+///////////////////////////////////////////////////////////////////////////////
+
+#include <boost/property_tree/ptree.hpp>
+
+using boost::property_tree::ptree;
+
+// Creates JSON file for unit test outputs
+class UnitTestJSON {
```

Comment:
I found this class confusing at first, because it's called "unit test" but doesn't actually represent a unit test. Instead it is used to build up a JSON output one piece at a time. Could we call it something like result_json_builder instead?